OverlayPlugin.Core/ClipboardTechSupport.cs

```diff
@@ -0,0 +1,156 @@
+// TODO: get Ravahn to expose more settings and include them
+// TODO: print warning on cactbot plugin / url / user dir mismatch
+// TODO: include first N lines of OverlayPlugin log
+
+namespace RainbowMage.OverlayPlugin
+{
+    using SimpleTable = List<List<string>>;
+
+    // Facilitates copying useful info to the clipboard about OP plugins and overlays
+    // so that folks don't have to paste screenshots in discord.
+    class ClipboardTechSupport
+    {
+        private SimpleTable plugins;
+        private SimpleTable overlays;
+        private SimpleTable settings;
+        private SimpleTable warnings;
+
+        public ClipboardTechSupport(TinyIoCContainer container)
+        {
+            var repository = container.Resolve<FFXIVRepository>();
+
+            warnings = new SimpleTable { new List<string> { "Warnings" } };
+            var opVersion = repository.GetOverlayPluginVersion();
+            if (opVersion < new Version(0, 19, 0))
+            {
+                warnings.Add(new List<string> { $"Out of date OverlayPlugin {opVersion.ToString()}, see https://overlayplugin.github.io/OverlayPlugin/fork_update.html" });
+            }
+
+
+            plugins = new SimpleTable { new List<string> { "Plugin Name", "Enabled", "Version", "Path" } };
+            foreach (var plugin in ActGlobals.oFormActMain.ActPlugins)
+            {
+                // TODO: plugin.pluginVersion has FileVersion, ProductVersion, etc, but all as a string.
+                // For now, ask FileVersionInfo to get this for us.
+                var fullPath = plugin.pluginFile.FullName;
+                var version = FileVersionInfo.GetVersionInfo(fullPath);
+                plugins.Add(new List<string>{
+                    plugin.pluginFile.Name,
+                    // TODO: could check plugin.pluginObj to see if it loaded successfully.
+                    plugin.cbEnabled.Checked.ToString(),
+                    version?.FileVersion?.ToString() ?? "",
+                    fullPath,
+                });
+            }
+
+            var pluginConfig = container.Resolve<IPluginConfig>();
+            overlays = new SimpleTable { new List<string> { "Overlay Name", "URL" } };
+            foreach (var overlay in pluginConfig.Overlays)
+            {
+                overlays.Add(new List<string>{
+                    overlay.Name,
+                    overlay.Url,
+                });
+            }
+
+            settings = new SimpleTable { new List<string> { "Various Settings", "Value" } };
+            settings.Add(new List<string> { "Plugin Language", repository.GetLanguage().ToString() });
+            settings.Add(new List<string> { "Machina Region", repository.GetMachinaRegion().ToString() });
+            // Only available when running the game.
+            string gameVersion = repository.GetGameVersion();
+            if (gameVersion != "")
+                settings.Add(new List<string> { "Game Version", gameVersion });
```

Comment:
Maybe this should note that the game's not running in the `settings` table?

Reply:
Done.